tailwind.config.js

```diff
@@ -181,6 +192,7 @@ module.exports = {
         success: "#E8FFE4",
         info: "#E4F5FF",
         error: "#FFE6E4",
+        "ztg-pink": "#DC056C",
```

Comment:
Can't see this used anywhere

Reply:
Ah yeah I think I added this on the wrong pr, it was meant to be on the new front page design pr